utils/src/main/java/software/amazon/awssdk/utils/TracingSystemSetting.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.utils;
+
+/**
+ * Tracing specific System Setting.
+ */
+public enum TracingSystemSetting implements SystemSetting {
```

Comment:
Let's make this package-private, leave it in src/main/java/software/amazon/awssdk/awscore/interceptor and mark it as an `@SdkInternalApi`. No need to publicize it or maintain it as a public API.